### PR TITLE
UX Changes

### PR DESCRIPTION
**Pull request by ron-zano**

### CODE REVIEW

File: apps/client/package.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
     "axios": "^1.7.2",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
-    "react-query": "^3.39.3"
+    "react-query": "^3.39.3",
```
Comment (seanyasno) [inline apps/client/package.json]: damn you're a boomer you're using the wrong version lol. After version 4 he migrated the naming from `react-query` to `@tanstack/react-query` and the library `react-query` is not longer maintained.

File: apps/client/src/pages/home/Home.tsx [hunk 2]
```diff
@@ -22,25 +23,24 @@ const Home: React.FC = () => {
...
-      />
-    </>
+      {isLoadingBasicPokemons ? (
+        <Loader />
+      ) : isNotNullOrUndefined(basicPokemonsError) ? (
+        <h1>{isAxiosError(basicPokemonsError) ? basicPokemonsError.message : 'unknown error'}</h1>
+      ) : (
+        <Pokedex
+          basicPokemons={filteredPokemons}
+          key={debouncedSearchTerm} // a key is needed in order to re-render the component
```
Comment (seanyasno) [inline apps/client/src/pages/home/Home.tsx]: You can remove this comment. We already understand that if you provide a key is to rerender to component when `debouncedSearchTerm` is changing

File: apps/client/src/pages/home/Home.tsx [hunk 3]
```diff
@@ -22,25 +23,24 @@ const Home: React.FC = () => {
...
-    );
-  }
-
   return (
-    <>
+    <div className="flex flex-col items-center min-w-[60vw] h-screen">
       <h1 className="text-center text-7xl p-5"> PokéRon</h1>
       <Search
         value={searchTerm}
         onChange={(e) => setSearchTerm(e.target.value)}
```
Comment (seanyasno) [inline apps/client/src/pages/home/Home.tsx]:
```suggestion
        onChange={(event) => setSearchTerm(event.target.value)}
```

File: apps/client/src/pages/home/Home.tsx [hunk 4]
```diff
@@ -22,25 +23,24 @@ const Home: React.FC = () => {
...
-      <Pokedex
-        basicPokemons={filteredPokemons}
-        key={debouncedSearchTerm} // a key is needed in order to re-render the component
-      />
-    </>
+      {isLoadingBasicPokemons ? (
+        <Loader />
+      ) : isNotNullOrUndefined(basicPokemonsError) ? (
+        <h1>{isAxiosError(basicPokemonsError) ? basicPokemonsError.message : 'unknown error'}</h1>
+      ) : (
```
Comment (seanyasno) [inline apps/client/src/pages/home/Home.tsx]: this is too much nested ifs and makes it hard to read...

**Review by seanyasno — APPROVED** [review]

File: apps/client/src/pages/home/Home.tsx (hunk 2 above)
Comment (ron-zano) [reply apps/client/src/pages/home/Home.tsx]: חחח הרגת אותי

**Pull request merged**